src/components/sectionList/listActions/SectionListActions.tsx

```diff
@@ -99,6 +105,24 @@ export const ActionMore = ({
                                 }}
                             ></MenuItem>
                         </TooltipWrapper>
+
+                        <TooltipWrapper
+                            condition={!deletable}
+                            content={TOOLTIPS.noEditAccess}
```

Comment:
You added a new Tooltip for deletion, but this still refers to edit.
Either remove delete tooltip and just use edit. Or update this reference.

Reply:
Oops, changed it to `TOOLTIPS.noDeleteAccess`.